gno.land/pkg/gnoclient/integration_test.go

```diff
@@ -556,7 +556,7 @@ func Echo(str string) string {
 	body2 := `package hello
 
 func Hello(str string) string {
-	return "Hello " + str + "!" 
```

Comment:
Aren't we testing the `make fmt` diff in the CI?

Reply:
It's a string literal, so `go fmt` (nor any variation like gofumpt) doesn't actually change it (as it would change the underlying behaviour of the code, so not being "just a formatter".)